kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -2391,9 +2423,9 @@ components:
             type: string
         content:
           type: string
-        keyFormat:
+        keyFormat: #deprecated - wont be filled - use 'keySerde' field instead
```

Comment:
`deprecated: true`?

Reply:
by some reason it is not allowed to place `deprecated: true` near the `ref`...